srcpkgs/php8.1/template

```diff
@@ -356,3 +382,130 @@ php8.1-ffi_package() {
 		vmove "usr/lib/php${_php_version}/modules/ffi.so"
 	}
 }
+
+php_package() {
+	depends="${sourcepkg}"
+	build_style=meta
+	short_desc="PHP scripting language meta package"
+	replaces="php-mcrypt<7.2.0"
+}
+
+php-devel_package() {
+	depends="${sourcepkg}-devel"
+	build_style=meta
+	short_desc="PHP development files meta package"
+}
+
+php-phpdbg_package() {
+	depends="${sourcepkg}-phpdbg"
+	build_style=meta
+	short_desc="PHP interactive debugger meta package"
+}
+
+php-cgi_package() {
+	depends="${sourcepkg}-cgi"
+	build_style=meta
+	short_desc="PHP CGI and FCGI SAPI meta package"
+}
+
+php-apache_package() {
+	depends="${sourcepkg}-apache"
+	build_style=meta
+	short_desc="PHP Apache SAPI meta package"
+}
+
+php-fpm_package() {
+	depends="${sourcepkg}-fpm"
+	build_style=meta
+	short_desc="PHP FastCGI Process Manager meta package"
```

Comment:
I think something like that could be helpful. Probably want a page since it's fairly easy to miss a short_desc if you just do `xi php-fpm`. Reminds me of https://github.com/void-linux/void-packages/issues/38306

Reply:
OK, I opened https://github.com/void-linux/void-docs/pull/731